modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectLayoutResourceImpl.java

```diff
@@ -259,6 +262,12 @@ private com.liferay.object.model.ObjectLayoutColumn _toObjectLayoutColumn(
 		ObjectField objectField = _objectFieldLocalService.fetchObjectField(
 			objectDefinitionId, objectLayoutColumn.getObjectFieldName());
 
+		if (objectField == null) {
+			throw new NoSuchObjectFieldException(
+				"There is no object field with the name: " +
+					objectLayoutColumn.getObjectFieldName());
+		}
```

Comment:
@samanzanedo Instead of throwing the exception here, could you please change from `fetchObjectField` to `getObjectField`.. The `getObjectField` method already throws the `NoSuchObjectFieldException`

Thanks!

Reply:
@samanzanedo change from `ObjectField objectField = _objectFieldLocalService.fetchObjectField(` to
`ObjectField objectField = _objectFieldLocalService.getObjectField(`, then you don't need to throw the NoSuchObjectFieldException..

Reply:
@carolmariaabb 

Done.
Thanks for the review!

Regards.